test/src/lightning.local.e2e.test.ts

```diff
@@ -1,27 +1,13 @@
 import { Lightning } from '@inco/js/lite';
-import { parseLocalEnv } from '@inco/js/local';
 import { anvil } from 'viem/chains';
 import { describe } from 'vitest';
 import { runE2ETest } from './lightning-test';
-import { loadDotEnv, readFileFromRoot } from './repo.ts';
 
 describe(`Lightning Local Node E2E`, { timeout: 50_000 }, async () => {
-  loadDotEnv();
-  const dumpParams = process.env.DUMP_ENV_FILE;
-  if (!dumpParams) {
-    throw new Error("DUMP_ENV_FILE doesn't exist!");
-  }
-  const buf = await readFileFromRoot(dumpParams);
-  const { SENDER_PRIVATE_KEY, ECIES_PUBLIC_KEY, EXECUTOR_ADDRESS } = parseLocalEnv(buf);
-  const zap = Lightning.custom({
-    executorAddress: EXECUTOR_ADDRESS,
-    eciesPublicKey: ECIES_PUBLIC_KEY,
-    chainId: BigInt(anvil.id),
-    covalidatorUrl: 'http://localhost:50055',
-  });
+  const zap = Lightning.localNode();
```

Comment:
This is all a client-side dev needs to do to get a fully functional local node testing SDK that is compatible with the docker image

Reply:
Nice